packages/itwin/breakdown-trees/src/Views/FunctionalityProviders/TreeNodeFunctionIconMapper.ts

```diff
@@ -73,15 +74,20 @@ export class TreeNodeFunctionIconInfoMapper {
   public async getFunctionIconInfosFor(node?: TreeModelNode): Promise<FunctionIconInfo[]> {
     const returnedList: number[] = [];
     if (node) {
-      const elementKey = this._treeDataProvider.getNodeKey(node.item);
-      if (NodeKey.isGroupingNodeKey(elementKey)) {
-        returnedList.push(...this._groupNodeFunctionIconInfos);
-      } else if (NodeKey.isInstancesNodeKey(elementKey)) {
-        const classHierarchyArray = await IModelReadRpcInterface.getClient().getClassHierarchy(this._treeDataProvider.imodel.getRpcProps(), elementKey.instanceKeys[0].className);
-        for (const className of classHierarchyArray) {
-          const mappedFunctionalityProviders = this._TreeNodeClassFunctionIconInfoMap.get(className);
-          if (mappedFunctionalityProviders)
-            returnedList.push(...mappedFunctionalityProviders);
+      if (isPresentationTreeNodeItem(node.item)) {
```

Comment:
nitpick - could instead use `node && isPresentationTreeNodeItem(node.item)` to reduce indentation